### PR TITLE
Udpate fn run doc

### PR DESCRIPTION
**Pull request by Shell32-Natsu**
Add a section for using imperative and declarative functions at the same time.

### CODE REVIEW

File: site/content/en/reference/fn/run/_index.md [hunk 1]
```diff
@@ -266,6 +266,12 @@ details in the following:
 
 * [Issue 757]
 
+#### Use Imperative and Declarative at the same time
```
Comment (frankfarzan) [inline site/content/en/reference/fn/run/_index.md:269]: Title is misleading because this is not allowed

**Comment by mortent** [comment]
Use the `docs:` prefix on the commit message(s) so we can filter out this from the release notes.

**Review by prachirp — CHANGES_REQUESTED** [review]

**Review by prachirp — APPROVED** [review]
LGTM. Can you move Scoping Rules section above Imperative Run Specifics?

**Comment by Shell32-Natsu** [comment]
@mortent @frankfarzan Please review.

**Review by frankfarzan — APPROVED** [review]

**Pull request merged**